db/migrate/20170711193304_remove_accepted_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAcceptedFromUsers < ActiveRecord::Migration
+  def change
+    remove_column :users, :accepted, :boolean
```

Comment:
You would not normally make a migration for this, since no one else has added the field yet, so you're just undoing what the branch does. You can use `rake db:rollback` to undo the migration on your machine, then just delete the migration.